scenarios/Assistants/api-in-a-box/math_tutor/assistant-math_tutor.ipynb

```diff
@@ -4,15 +4,63 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Mart Tutor Assistant\n",
+    "# Math Tutor Assistant"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Objective\n",
+    "\n",
+    "{{TODO: Add a one or two sentences describing the learning outcomes for this notebook}}\n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- {{TODO: Add short descriptions and links to documentation for Azure AI services used}}\n",
     "\n",
     "Reference:\n",
     "\n",
     "- [Assistants OpenAI Overview](https://platform.openai.com/docs/assistants/overview)\n",
     "- [Github-OpenAI Python/examples/Assistant demo notebook](https://github.com/openai/openai-python/blob/main/examples/assistant.py)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## About this example\n",
+    "\n",
+    "{{TODO: Include a paragraph or two explaining what this example demonstrates, who should be interested in it, and what you need to know before you get started.}}\n",
+    "\n"
+   ]
```

Comment:
Needs a description.

Reply:
## About this example

This sample shows users how to create an Azure OpenAI Assistant named "Math Tutor" using the Azure OpenAI API. The assistant is designed to function as a personal math tutor, capable of answering math questions through code interpretation. The script initiates a conversation with the assistant, guiding it through various mathematical queries and scenarios to showcase its capabilities.

This sample provides developers with a clear demonstration of how to leverage the core concepts of the Assistants API into their projects, highlighting its simplicity and effectiveness in leveraging foundational concepts.

Reply:
Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/6118b1a20eeae2209d8b9442306d9a769b55096a